frontend/src/lib/b6.ts

```diff
@@ -32,7 +32,6 @@ const stack = async (request: UIRequestProto & { logEvent: Event }) => {
 
 const startup = async (urlParams: {
     z: string;
-    ll: string;
```

Comment:
By completely removing the `ll` parameter on the startup request, we'd lose the ability to share a URL to a specific location. I've changed this to make it an optional parameter instead.